Makefile

```diff
@@ -309,9 +309,8 @@ install/lib/libredis++.a:
 	@cd third-party/redis-plus-plus && \
 	mkdir -p compile && \
 	cd compile && \
-	cmake -DCMAKE_BUILD_TYPE=Release -DREDIS_PLUS_PLUS_BUILD_TEST=OFF -DREDIS_PLUS_PLUS_BUILD_SHARED=OFF -DCMAKE_PREFIX_PATH="../../../install/lib/" -DCMAKE_INSTALL_PREFIX="../../../install" -DCMAKE_CXX_STANDARD=17 .. && \
-	CC=gcc CXX=g++ make -j && \
```

Comment:
Does deleting this line work with the other compilers?

Reply:
Yes, I think the intention was always to use the GCC compilers (see old line 313).